### DISCUSSION TITLE
admin panel works, shortlinks error too many redirects

### DISCUSSION THREAD

**Discussion by adnaanqureshi** [YOURLS/YOURLS#3503] [Q&A] [answered]
Hi
Ive used yourls before, but added a new install and its on cloudflare - which potentially is the cause of my troubles.
the cloudflare dns has just an A record for root to the true server ip

Im getting too many redirects error. tried clearing cookies / caches etc

the domain is https://poppylink.co 
nothing else is installed on this domains document root 

server OP is AlmaLinux v8.7.0

the admin panel works absolutely fine at https://poppylink.co/admin/ -  no errors on YOURLS_DEBUG

the links are created like this
https://poppylink.co/8hEFB (clicking this link will demonstrate the error)

No links work - I get too many redirects error.

A redirect checker shows:
![image](https://user-images.githubusercontent.com/85097710/216176476-85007675-7145-4470-8c5b-3725f84786af.png)

The yourls url db entry looks perfectly fine. 

The config.php has this url setting:

'YOURLS_SITE', 'https://poppylink.co'

Cloudflare - ive tried SSL off, flexible, full, and full strict. nothing makes the links work.
SSL checker doesnt find any issue.

the htaccess is:
![image](https://user-images.githubusercontent.com/85097710/216178616-2e1ec6ba-a127-4ac8-8e6c-2b4f8184d831.png)

the funny thing is the redirect checker shows the shortlink redirecting to shortlink/longurl (see photo)

Any idea on why this is happening?

ps the demo short links that come with yourls seems to work fine !!!

**Comment by dgw** [comment, marked as answer]
Long URL is missing the protocol, I'd bet.

**Reply by adnaanqureshi** [reply]
OOOOH!!!
yes!
